scripts/deploy-rETH-THETA-stakingrewards.js

```diff
@@ -1,24 +1,72 @@
 const hre = require("hardhat");
 const { STAKING_REWARDS_rETHTHETA_PARAMS } = require("../params");
+const moment = require("moment");
 
 async function main() {
-  // We get the contract to deploy
+  const [deployer] = await ethers.getSigners();
+  const network = hre.network.name;
+
+  /** We get the contract to deploy */
   const RibbonStakingRewards = await hre.ethers.getContractFactory(
     "StakingRewards"
   );
+
+  /** Set parameter */
+  const owner =
+    network === "kovan"
+      ? deployer.address
+      : STAKING_REWARDS_rETHTHETA_PARAMS.OWNER;
+  const rewardsDistAddress =
+    network === "kovan"
+      ? deployer.address
+      : STAKING_REWARDS_rETHTHETA_PARAMS.REWARDS_DIST_ADDR;
+  /** Kovan will be $KRBN */
+  const rewardsToken =
+    network === "kovan"
+      ? "0x567e482AF973187648Af9FC56d2Caec212c1CAca"
+      : STAKING_REWARDS_rETHTHETA_PARAMS.REWARDS_TOKEN;
+  const stakingToken =
+    network === "kovan"
+      ? "0x5B8E6eaB6502CC642d00A55F0d8B5f5557c94Bc5"
+      : STAKING_REWARDS_rETHTHETA_PARAMS.STAKING_TOKEN;
+  /**
+   * Kovan will always be set to next saturday 1200 UTC
+   */
+  const startEmission =
+    network === "kovan"
+      ? moment()
+          .startOf("isoWeek")
+          .add(1, "week")
+          .utc(true)
+          .day("saturday")
+          .unix()
+      : STAKING_REWARDS_rETHTHETA_PARAMS.START_EMISSION;
```

Comment:
Wouldn't you want to set the `startEmission` to be now? Because you want to test it immediately instead of waiting a week